### DISCUSSION TITLE
what is the narrow field interested

### DISCUSSION THREAD

**Discussion by Faisal-Hajari** [Faisal-Hajari/KD#2] [Q&A]
I want to look into prompting and working on the non contrastive loss function for the CLIP models.<br/> 

[Learning to Name Classes for Vision and Language Models](https://openaccess.thecvf.com/content/CVPR2023/papers/Parisot_Learning_To_Name_Classes_for_Vision_and_Language_Models_CVPR_2023_paper.pdf)

**Comment by SARA-SULAIMAN** [comment]
papers i'm interested in :

- WinCLIP: Zero-/Few-Shot Anomaly Classification and Segmentation
- BiCro: Noisy Correspondence Rectification for Multi-modality Data via Bi-directional Cross-modal Similarity Consistency
- Multi-level Logit Distillation
-
